README.md

```diff
@@ -33,4 +33,5 @@ We use MarkdownLint to lint the markdown files. This runs as a test on GitHub ac
 ```bash
 yarn
 yarn lint:md
+yarn lint-fix:prettier
```

Comment:
Let's discuss this in another PR. @Kalaiselvan88, can you remove this please so that I can merge?

The reason is that this is misleading as it is under the wrong title.

Reply:
done